### ISSUE TITLE
Bug: Nothing renders on mac

### ISSUE THREAD

**Issue by rexfleischer** [SpaiR/imgui-java#288]
### Version

1.87.5

### What happened?

I upgraded to the newest version and nothing from ImGui renders. If I revert to version 1.86.12, everything seems to work normally.

My machine is Apple M1 Max, so probably related to mac weirdness..?

### Reproduction

I can reliably reproduce this by switching between 1.87.5 and 1.86.12 versions.

Let me know if you need someone to test changes on an Apple M1 Max.

**Comment by SpaiR** [comment]
Could you please double-check the functionality using the Example module of the binding? The reason I ask is that I also have a macOS M1 laptop, and I do all my development on it with tests on other platforms. I haven't experienced any rendering issues on either x86 or ARM builds. Perhaps you're using a custom renderer or something along those lines?

**Comment by rexfleischer** [comment]
Yep, you're correct in asking for this, the example app works on main of this repo. So, probably some mixture of my setup being stupid and therefore not compatible with some configuration thing. 

Sorry for the false alarm!
